templates/families/batteries.json

```diff
@@ -0,0 +1,507 @@
+{
+  "code": "batteries",
+  "labels": { "en_US": "Batteries" },
+  "description": {
+    "en_US": "Power up your devices with our 'Batteries' DPP template - effortlessly manage your battery inventory, ensuring uninterrupted performance and endless productivity whenever you need it most."
+  },
+  "attribute_as_main_media": "main_image",
+  "attribute_as_label": "product_title",
+  "attributes": [
+    {
+      "code": "EAN",
+      "labels": { "en_US": "Main Identifier" },
+      "type": "pim_catalog_identifier",
+      "scopable": false,
+      "localizable": false,
+      "unique": true
+    },
+    {
+      "code": "product_title",
+      "labels": { "en_US": "Product title" },
+      "type": "pim_catalog_text",
+      "scopable": false,
+      "localizable": true,
+      "unique": false
+    },
+    {
+      "code": "main_image",
+      "labels": { "en_US": "Main image" },
+      "type": "pim_catalog_image",
+      "scopable": false,
+      "localizable": false,
+      "unique": false
+    },
+    {
+      "code": "battery_passport_identification",
+      "labels": { "en_US": "Battery passport identification" },
+      "type": "pim_catalog_identifier",
+      "scopable": false,
+      "localizable": false,
+      "unique": true
+    },
+    {
+      "code": "responsible_economic_operator_identification",
+      "labels": { "en_US": "Responsible economic operator identification" },
+      "type": "pim_catalog_number",
+      "scopable": false,
+      "localizable": false,
+      "unique": false,
+      "decimals_allowed": false,
+      "negative_allowed": false
+    },
+    {
+      "code": "manufacturers_identification",
+      "labels": { "en_US": "Manufacturer\u2019s identification" },
```

Comment:
Maybe an encoding problem, not really sure that it will be displayed well on the PIM